integration_tests/src/gstreamer.rs

```diff
@@ -0,0 +1,280 @@
+use anyhow::{anyhow, Result};
```

Comment:
The same as ffmpeg:
- make everything non-pub that does not have to be
- `start_gst_send_sample_tcp` name looks like some internal part of `start_gst_send_tcp`
  - rename `start_gst_send_sample_tcp` to  `start_gst_sent_tcp`
  - rename other function (similar to suggestions from ffmpeg)